scripts/tests/ledger_test_apply.sh

```diff
@@ -0,0 +1,42 @@
+#!/bin/bash
+
+# test mina ledger test-apply
+set -x
+
+export MINA_PRIVKEY_PASS='naughty blue worm'
+
+MINA_APP=_build/default/src/app/cli/src/mina.exe
+RUNTIME_LEDGER_APP=_build/default/src/app/runtime_genesis_ledger/runtime_genesis_ledger.exe
+
+TEMP_FOLDER=$(mktemp -d)
+ACCOUNTS_FILE=$TEMP_FOLDER/accounts.json
+GENESIS_LEDGER=$TEMP_FOLDER/genesis_ledger.config
+SENDER=$TEMP_FOLDER/sender
+
+while [[ "$#" -gt 0 ]]; do case $1 in
+  -m|--mina-app) MINA_APP="$2"; shift;;
+  -r|--runtime-ledger-app) RUNTIME_LEDGER_APP="$2"; shift;;
+  *) echo "Unknown parameter passed: $1"; exit 1;;
+esac; shift; done
+
+
+echo "Exporting ledger to $TEMP_FOLDER"
+$MINA_APP ledger test generate-accounts -n 2 --min-balance 100000 --max-balance 1000000 > $ACCOUNTS_FILE
```

Comment:
I don't think ledger of size `2` is good for a test.

Let's set it to `200k`.
Caveat: generating such a ledger and running `runtime_genesis_ledger.exe` against it might be a very long endeavor. That's why I suggested to use a pre-generated ledger from s3.

Ledger with two entries is just too different from one on mainnet's network, different enough that I wouldn't count it as a good test.

Reply:
Good catch! I forgot to generate more accounts! I will set it to 200k and see how much time is needed to finish test. If this is too long I will used s3 bucket. From my experience having pre-generated test data is always problematic from merges perspective problem and couching others how to generate it.  I will post my findings